app/controllers/concerns/idv/phone_otp_rate_limitable.rb

```diff
@@ -42,7 +42,6 @@ def handle_max_attempts(type)
         type,
         decorated_user,
       )
-      sign_out
```

Comment:
Wanted to bring attention to this, as it's more of a logic change than I might have hoped for to get the newly-JS-enabled specs to pass. There's more detail in the extended commit description of dfe18a3. This also impacts real-world production users, who will see the Sign In page instead of the Account Locked page while throttled. I don't really know that it's crucial that we sign out the user when they're throttled here? But maybe I'm missing something. 

cc @jmhooper (via git blame'd #2430)

Reply:
Okay, I've applied some brainpower to this problem. I think signing users out here is the wrong move. We def wanna sign them out on MFA throttles, but users who are throttled during proofing can still sign in for non-proofed contexts.

tl;dr removing the sign out is an advancement and not a regression